packages/shared/locales/en.json

```
@@ -711,7 +711,8 @@
                     "primary": "Primary key",
                     "setAsPrimary": "Set as primary",
                     "unsetAsPrimary": "Unset as primary"
-                }
+                },
+                "hint": "Your main account has a negative Block Issuance Credit (BIC). Please select another account as the main one."
```

```suggestion
                "hint": "Your main account has a negative (BIC). Please select another account as the main one."
```
`Block Issuance Credit` is too large imo

---

i think it's a good idea to be clear about it :pray:  the notification already mentions BIC, and honestly, I don't see the length being a problem.. :see_no_evil:

---

okeey 👌🏼